pentaho/src/main/resources/solution/bi-developers/cda/cdafiles/hci.cda

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CDADescriptor>
+
+    <!-- DataSource definition. 
+	
+	Type controls what the datasource type is.
+	The connection and query controls how the data is fetched. These values
+	are specific to each access type
+
+	-->
+    <DataSources>
+        <Connection id="1" type="hds.hci">
+            <Url>http://10.76.17.150:8888/api/search/query</Url>
```

Comment:
If you can edit this with cda, then treat this as an default value. Use 127.0.0.1 for example.